### DISCUSSION TITLE
请问如何设置隐藏目录？

### DISCUSSION THREAD

**Discussion by woshichenghaibo** [icret/EasyImages2.0#205] [问答] [answered]
我在图床安全里面设置了隐藏目录，但是上传去的图片地址还是显示我的真实目录

**Comment by icret** [comment, marked as answer]
https://icret.github.io/EasyImages2.0/#/./%E9%9A%90%E8%97%8F%E5%AD%98%E5%82%A8%E8%B7%AF%E5%BE%84

**Comment by woshichenghaibo** [comment]
没有nginx如何绑定域名到目录？

________________________________
发件人: icret ***@***.***>
发送时间: 2024年10月18日 3:57
收件人: icret/EasyImages2.0 ***@***.***>
抄送: woshichenghaibo ***@***.***>; Author ***@***.***>
主题: Re: [icret/EasyImages2.0] 请问如何设置隐藏目录？ (Discussion #205)


https://icret.github.io/EasyImages2.0/#/./%E9%9A%90%E8%97%8F%E5%AD%98%E5%82%A8%E8%B7%AF%E5%BE%84

―
